static/js/services/enketo.js

```diff
@@ -24,47 +23,30 @@ angular.module('inboxServices').service('Enketo',
     var FORM_ATTACHMENT_NAME = 'xml';
     var REPORT_ATTACHMENT_NAME = this.REPORT_ATTACHMENT_NAME = 'content';
 
-    var replaceJavarosaMediaWithLoaders = function(formInternalId, form) {
```

Comment:
I don't fully understand the changes made to this function.  Will media loaders, and the resulting loaded-media, now be cached?